features/step_definitions/timehelpers_steps.rb

```diff
@@ -1,16 +1,21 @@
 # frozen_string_literal: true
 
 After do
+  Time.zone = 'UTC'
```

Comment:
1. instead of hard-coding `UTC` I'd use `Rails.application.config.time_zone` (just in case we want to use another time zone at some point - I hope not, though!)

2. Maybe it's more logical to do set it in `Before`. But maybe not :grimacing:  It's probably just fine here

Reply:
I applied this. I don't think there's a difference between `Before` and `After` in practice.